ci/pipeline.yml

```diff
@@ -96,10 +96,10 @@ jobs:
             dir: src
             path: ci/tasks/deploy.sh
         params:
-          <<: *staging-cf
-          CF_APP_NAME: pages-builder-staging
+          <<: *cf
+          CF_APP_NAME: pages-builder-((env))
           CF_MANIFEST: .cloudgov/manifest.yml
-          CF_VARS_FILE: .cloudgov/vars/pages-staging.yml
+          CF_VARS_FILE: .cloudgov/vars/pages-((env)).yml
```

Comment:
need to make sure we follow this naming convention for the vars then, or could use the filepath to do this to, ie `/.cloudgov/vars/<env>/vars.yml`

Reply:
@apburnes any cloud.gov convention or preference here?

Reply:
No. Dave started the `.cloudgov/vars/*` convention since for our CF apps. CG deployments are mostly not CF apps so there hasn't been a defined convention for them.